kissnet.hpp

```diff
@@ -119,6 +120,9 @@
 #include <string>
 #include <utility>
 
+#include <openssl\ssl.h>
+#include <openssl\err.h>
```

Comment:
Paths with anti-slashes only works with Microsoft Windows. That's a big nono. Also these includes needs to be in the `#ifdef` for openssl!

Reply:
Actually in latest version it's in the #ifdef... But still, don't use antislashes, ever! 😉